docs/markdown/Contributing.md

```diff
@@ -380,6 +378,30 @@ matched:
 | `literal` | Literal match (default) |
 | `re`      | regex match             |
 
+#### skip_on_env
+
+The `skip_on_env` key can be used to specify a list of environment variables. If
+at least one environment variable in the `skip_on_env` list is present, the test
+is skipped.
+
+#### skip_on_jobname
+
+The `skip_on_jobname` key contains a list of strings. If the `MESON_CI_JOBNAME`
+environment variable is set, and any of them are contained in it, the test is
```

Comment:
This should probably make it clearer that only a substring has to be matched, because I missed that part initially.

Reply:
Yeah, I'll adjust that text.

I did initally think of making the jobnames a bit more structured (e.g. a series of '-' separated strings, and exactly matching one of those, so e.g. foo and nofoo were matchable), but that didn't seem to be needed yet.